cdmtaskservice/mongo.py

```diff
@@ -105,6 +110,17 @@ async def get_image(self, name: str, digest: str | None = None, tag: str | None
             raise NoSuchImageError(f"No image {name} {err} exists in the system.")
         return models.Image.model_construct(**doc)
 
+    async def save_job(self, job: models.Job):
+        """ Save a job. Job IDs are expected to be unique."""
+        _not_falsy(job, "job")
+        jobd = job.model_dump()
+        jobi = jobd[models.FLD_JOB_JOB_INPUT]
```

Comment:
```suggestion
        jobi = jobd.get(models.FLD_JOB_JOB_INPUT, {})
```

Can we assume the job input always exists at this stage?

Reply:
Yes, it's a required part of a Job